pvr.teleboy/resources/language/resource.language.de_de/strings.po

```diff
@@ -0,0 +1,66 @@
+# Kodi Media Center language file
+# Addon Name: pvr.teleboy
+# Addon id: pvr.teleboy
+# Addon Provider: rbuehlma
+
+msgid ""
+msgstr ""
+
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: de_DE\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. Setting to set used stream username
+#: pvr.zattoo/resources/settings.xml
+msgctxt "#30000"
+msgid "Username"
+msgstr "Benutzername"
+
+#. Setting to set used stream password
+#: pvr.zattoo/resources/settings.xml
+msgctxt "#30001"
+msgid "Password"
+msgstr "Passwort"
+
+#. Boolean setting to use favourite streams only
+#: pvr.zattoo/resources/settings.xml
+msgctxt "#30002"
+msgid "Favourite channels only"
+msgstr "Nur favorisierte Kanäle nutzen"
+
+#. Help text to setting #30002
+#: pvr.zattoo/resources/settings.xml
+msgctxt "#30003"
+msgid "Load only channels which are in the favourite list."
+msgstr "Lädt nur Kanäle welche in favorisierter Liste enthalten sind."
+
+#. Boolean setting to enable dolby stream usage
+#: pvr.zattoo/resources/settings.xml
+msgctxt "#30004"
+msgid "Enable Dolby"
+msgstr "Aktiviere Dolby"
+
+#. Help text to setting #30010
+#: pvr.zattoo/resources/settings.xml
```

Comment:
You was a bit to fast :sweat_smile:, seen that something was gone wrong at home and here was come back my previous changed zattoo to teleboy, also was two added german strings was gone away,

Will create another request to fix.